packages/google-types/src/nodes/df-es/image.ts

```diff
@@ -0,0 +1,16 @@
+import { BaseNode, BaseStep } from '../types';
+import { NodeType } from './types';
+
+export type StepData = {
+  imageUrl: string;
+};
```

Comment:
this could be an interface, not sure if that's a convention we actually use though